config/index.js

```diff
@@ -541,6 +541,14 @@ var conf = convict({
       }
     }
   },
+  serverVerification: {
+    key: {
+      default: 'YOU MUST CHANGE ME',
+      doc: 'server verification key, as used by content-server',
+      format: String,
+      env: 'SERVER_VERIFICATION_KEY'
```

Comment:
Good call; we already call it "clientAddress" in other places in the code so that should like a good option to me, we're "overriding the client address".